deliver/lib/deliver/runner.rb

```diff
@@ -72,10 +72,10 @@ def upload_metadata
     def prepare_app_icons(options = {})
       return unless options[:metadata_path]
 
-      default_app_icon_path = File.join(options[:metadata_path], "app_icon.png")
+      default_app_icon_path = Dir["#{options[:metadata_path]}/app_icon.{png,jpg}"].first
```

Comment:
the thing is if we downloaded it and respected the original ending, we may have a `.jpg` in here.

you are right only `.first` will be used, what do you think is a good solution for that edge case (user downloaded .jpg, and put a .png in his metadata folder)